week1/index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css">
+    <title>help hamsick</title>
+  </head>
+
+  <body>
+    <header>
+      <div class="headerColorBox">
+      <h1>💛🍔 햄식이를 도와줘 🍔💛</h1>
+      </div>
+    </header>
+
+    <main>
+      <aside class="sidebar">
+        <section class="option_card">
+          <h2 class="optionTitle">다른거 먹자</h2>
+          <div class="option"> 
+            <p class="macdonalds">맥도날드</p>
+            <p class="burgerking">버거킹</p>
+          </div>
+        </section>
+      </aside>
+      <section class="burger">
+        <article class="burger_card">
+          <img src="https://www.mcdonalds.co.kr/upload/product/pcfile/1583727855319.png" alt="빅맥사진">
+          <div class="burger_content">
+            <h2 class="buger_name">빅맥</h2>
+            <div class="burger_price">5,500</div>
```

Comment:
가격정보도 p테그를 이용하는 것이 좋을 것 같아요! 
```suggestion
            <p class="burger_price">5,500</p>
```

Reply:
@Nahee-Park 대박꼼꼼해><<